### PR TITLE
removed workaround for OSD timeout after upgrade

### PR DESCRIPTION
**Pull request by AviadP**
with this BZ https://bugzilla.redhat.com/show_bug.cgi?id=1840729 moved to verify, im removing the workaround we had in the code, time out set to 10 minutes again

Signed-off-by: aviadp <apolak@redhat.com>

### CODE REVIEW

**Review by ebenahar — APPROVED** [review]

File: ocs_ci/ocs/ocs_upgrade.py [hunk 1]
```diff
@@ -110,12 +110,9 @@ def verify_image_versions(old_images, upgrade_version):
     )
     verify_pods_upgraded(old_images, selector=constants.MON_APP_LABEL, count=3)
     verify_pods_upgraded(old_images, selector=constants.MGR_APP_LABEL)
-    # OSD upgrade have timeout 10mins for new attempt if cluster is not health.
-    # https://bugzilla.redhat.com/show_bug.cgi?id=1840729 setting timeout for
-    # 12.5 minutes per OSD
     verify_pods_upgraded(
         old_images, selector=constants.OSD_APP_LABEL, count=osd_count,
-        timeout=750 * osd_count,
+        timeout=600 * osd_count,
```
Comment (petr-balogh) [inline ocs_ci/ocs/ocs_upgrade.py:115]: what about upgrade from 4.3 to 4.4.  If we need to still run that, then we can maybe have an issue and maybe we should change this value only when version >= 4.5

**Review by petr-balogh — COMMENTED** [review]
This has to be verified with all pre/post upgrade tests together @AviadP + enable parameter RUN IO IN BACKGROUND.  Please do not merge till it's verified like this please.

File: ocs_ci/ocs/ocs_upgrade.py (hunk 1 above)
Comment (AviadP) [reply ocs_ci/ocs/ocs_upgrade.py:115]: i can add it as constant and use it according to upgraded version

Comment (petr-balogh) [reply ocs_ci/ocs/ocs_upgrade.py:115]: Please do @AviadP

**Review by petr-balogh — APPROVED** [review]

**Review by wusui — APPROVED** [review]
LGTM

**Review by ebenahar — APPROVED** [review]

**Pull request merged**